package-scripts.js

```diff
@@ -15,18 +17,26 @@ const config = require('./dist/config.js').config;
 /**
  * environment variables you can use to configure stuff like migrateContracts
  */
-const pathArcJs = env.pathArcJs || cwd;
-const pathArcJsContracts = env.pathArcJsContracts || "./contracts";
-const pathDaostackArcRepo = env.pathDaostackArcRepo || "../daostack";
+const pathArcJsRoot = env.pathArcJsRoot || cwd;
+
+const pathArcJsContracts =
+  env.pathArcJsContracts || joinPath(pathArcJsRoot, "contracts");
+
+const pathDaostackArcRepo =
+  env.pathDaostackArcRepo ||
+  joinPath(pathArcJsRoot, "node_modules/daostack-arc");
+
 const pathDaostackArcRepoContracts =
   env.pathDaostackArcRepoContracts ||
   joinPath(pathDaostackArcRepo, "build/contracts");
-const pathDaostackArcTestrpcDb = "./testrpcDb";
-const pathDaostackArcTestrpcDbZip = "./testrpcDb.zip";
+
+const pathDaostackArcTestrpcDb = joinPath(pathArcJsRoot, "testrpcDb");
+const pathDaostackArcTestrpcDbZip = joinPath(pathArcJsRoot, "testrpcDb.zip");
+
 const network = config.get('network');
 
 // "console.log(`cwd: ${cwd}`)"
-// console.log(`network: ${network ? network: "testrpc"}`)
+// console.log(`network: ${network ? network: "ganache"}`)
```

Comment:
please remove commented code

Reply:
I agree, I dont like to leave comments around